modules/verifying-serverless-app-deployment.adoc

```diff
@@ -24,3 +31,34 @@ helloworld-go   http://helloworld-go.default.example.com   helloworld-go-4wsd2
 $ curl http://helloworld-go.default.example.com
 Hello World: Go Sample v1!
 ----
++
+----
```

Comment:
At first I thought this was an accidental duplication, but now I see that it's https vs. http. Maybe consider adding a heading to the code block to identify the difference, like `.Example HTTP request` and `.Example HTTPS request` or something.